firebase.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Remove this file and dataObjectChrome.png.

Reply:
done